modAionImpl/src/org/aion/zero/impl/AionBlockchainImpl.java

```diff
@@ -576,8 +576,15 @@ ImportResult tryToConnectInternal(final AionBlock block, long currTimeSeconds) {
 
         if (block.getTimestamp()
                 > (currTimeSeconds
-                        + this.chainConfiguration.getConstants().getClockDriftBufferTime()))
+                        + this.chainConfiguration.getConstants().getClockDriftBufferTime())) {
+            if (LOG.isDebugEnabled()) {
+                LOG.debug(
+                        "Block {} invalid due to timestamp {}.",
+                        block.getShortHash(),
```

Comment:
Suggest using full hash here, makes everyones lives easier for a little more verbosity